issues/disruption-probe.md

```diff
@@ -0,0 +1,66 @@
+# Feature request / problem statement
+
+Our users rely heavily on eviction and pod disruption budgets for stability and for fleet management, see [this previous talk on rotating nodes](https://www.youtube.com/watch?v=KQ1obaC-ht0) for more context. TODO any other talks worth showing?
+
+An issue our users often run into is that temporarily they do not want their pod to be evicted. The readiness probe is not an option because the pods critically do still need to serve traffic. TODO small justification / real example internal and external (elastic search).
+
+We have worked around this writing a controller that modifies the `spec.maxUnavailable` field, setting it to `0` to block disruption and setting it back to enable.
+ 
+The request is to have a mechanism provided by Kubernetes that can distinguish between whether a pod should be routable (readiness) and whether a pod should be disruptable. The solution outlined here is to have a disruption probe and pod status, similar to readiness.
```

Comment:
Should we mention that it should be optional?  Maybe something like, "If not provided, the readiness probe would still cover both cases"?